[BUG] Unknown CMake command in CMakeLists.txt in unit test "Commands"

**Describe the bug**
CMake Error at UnitTests/Schemas/IFC4X1/Comments/CMakeLists.txt:20 (CreateIfcFileUnitTestForSchema):
 Unknown CMake command "CreateIfcFileUnitTestForSchema".

**To Reproduce**
Steps to reproduce the behavior:
1. Fetch all new changes
2. Start configuration process
3. See error

**Expected behavior**
CMake should configure without error

**CMake configuration**
If applicable, list any special configurations you have selected in CMake.
- EARLYBINDING_WITH_IFC4x1: selected

--- a/UnitTests/Schemas/IFC4X1/Comments/CMakeLists.txt
+++ b/UnitTests/Schemas/IFC4X1/Comments/CMakeLists.txt
@@ -17,4 +17,4 @@
 
 include(CreateUnitTests)
 
-CreateIfcFileUnitTestForSchema(Comments IFC4X1)
+CreateIfcFileVisualUnitTestForSchema(Comments IFC4X1)

